guides/customize_tinymce.md

```diff
@@ -73,6 +73,40 @@ That's it. Now the editor can format a block of text as headline 2. Nice!
 Custom buttons always appear in all richtext editors in all elements in Alchemy.
 :::
 
+## Adding editor plugins
+It’s possible to configure the editor with plugins.
```

Comment:
I think it reads better if we write 

It is possible to extend the tinymce editor with plugins.